scripts/data_contamination/load_documents.py

```diff
@@ -0,0 +1,139 @@
+import json
+
+from typing import Iterator
+
+
+def get_document_iterator(file_path: str, file_format: str) -> Iterator[str]:
```

Comment:
Could consider something smarter/nicer for all the file parsing with all `elif` but not sure it's worth it at the moment

Reply:
I'm okay with this.